Tests are failing for me as well - I'm currently not in a position to dig deeper, so this might have to wait, sorry.

```
E    assert '--- \n\n+++ ...rem foo ipsum' == '---  \n\n+++ ...rem foo ipsum'
E      - --- 
E      + ---  
E      ?     +
E        
E      - +++ 
E      + +++  
E      ?     +
E        
E      - @@ -1 +1 @@
E      + @@ -1,1 +1,1 @@
E      ?      ++   ++
E        
E        -lorem ipsum
E        +lorem foo ipsum
```

--- a/differ/test/test_differ.py
+++ b/differ/test/test_differ.py
@@ -13,7 +13,7 @@ def test_diff():
 	assert actual == expected
 
 	actual = diff(a, b, "unified")
-	expected = "---  \n\n+++  \n\n@@ -1,1 +1,1 @@\n\n-lorem ipsum\n+lorem foo ipsum"
+	expected = "--- \n\n+++ \n\n@@ -1 +1 @@\n\n-lorem ipsum\n+lorem foo ipsum"
 	assert actual == expected
 
 	actual = diff(a, b, "horizontal")

